lib/auth0/api/authentication_endpoints.rb

```diff
@@ -49,6 +51,30 @@ def exchange_auth_code_for_tokens(
         AccessToken.from_response post('/oauth/token', request_params)
       end
 
+      # Get access and ID tokens using a refresh token.
+      # @see https://auth0.com/docs/api/authentication#refresh-token
+      # @param refresh_token [string] Refresh token to use. Request this with
+      #   the offline_access scope when logging in.
+      # @param client_id [string] Client ID for the Application
+      # @param client_secret [string] Client Secret for the Application.
+      #   Required when the Application's Token Endpoint Authentication Method
+      #   is Post or Basic.
+      # @return [AccessToken] Returns tokens allowed in the refresh_token
+      def exchange_refresh_token(
+        refresh_token,
+        client_id: @client_id,
+        client_secret: @client_secret
+      )
+        raise Auth0::InvalidParameter, 'Must provide a refresh token' if refresh_token.to_s.empty?
+        request_params = {
+          grant_type:    'refresh_token',
+          client_id:     client_id,
+          client_secret: client_secret,
+          refresh_token:  refresh_token
```

Comment:
Alignment is off here. May be easiest to pull them all to 1 space from the colon?